portfolio/WEB-INF/appengine-generated/datastore-indexes-auto.xml

```diff
@@ -0,0 +1,4 @@
+<!-- Indices written at Wed, 15 Jul 2020 17:07:51 UTC -->
```

Comment:
local_db.bin & datastore-indexes-auto.xml probably shouldn't be in the PR. Can you remove them? Also, I feel like .bin files should be excluded via .gitignore file. Are they not?